units/SDL_properties.inc

```diff
@@ -297,10 +297,10 @@ function SDL_SetFloatProperty(props: TSDL_PropertiesID; name: PAnsiChar; value:
  *
  * \since This function is available since SDL 3.1.3.
  *
- * \sa SDL_GecbooleanProperty
+ * \sa SDL_GetBooleanProperty
   }
-function SDL_SecbooleanProperty(props: TSDL_PropertiesID; name: PAnsiChar; value: cbool): cbool; cdecl;
-  external SDL_LibName {$IFDEF DELPHI} {$IFDEF MACOS} name '_SDL_SecbooleanProperty' {$ENDIF} {$ENDIF};
+function SDL_SetBooleanProperty(props: TSDL_PropertiesID; name: PAnsiChar; value: Boolean): Boolean; cdecl;
+  external SDL_LibName {$IFDEF DELPHI} {$IFDEF MACOS} name '_SDL_SetBooleanProperty' {$ENDIF} {$ENDIF};
```

Comment:
An additional bugfix made along the way - the `SDL_SetBooleanProperty` function was named `SDL_SecboolProperty` in our code. Looks like a victim of a search & replace.